alembic/env.py

```diff
@@ -39,6 +39,7 @@
 from api.events.models import Events
 from api.question.models import Question
 from api.response.models import Response
+from api.tag.models import Tag
```

Comment:
Great work @kampkelly  👍 .

Can you share with us why this line was included?

Reply:
I am importing the tag model here so alembic can generate migrations for the tags whenever there is a change in the model.